tools/vscode/ddlog-language/syntaxes/ddlog.tmLanguage.json

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
is this file generated by some tools? If so it should not be in the repository but in .gitignore.